### ISSUE TITLE
Update README template regarding users

### ISSUE THREAD

**Issue by dacur** [smashingboxes/boxcar-generator-archive#148]
The [template](https://github.com/smashingboxes/boxcar/blob/d93d399d72ca242b0d55a6ed7999534e743f7c0e/templates/README.md.erb#L43) says:

```
### Seeded logins

For a list of credentials you can use to log into the app, check [the seeds file](db/seeds.rb).
These logins are created via `rails db:setup` above.
```

There aren't any credentials being made, unless I'm missing something.  This should be updated, removed, or the seeds file should create a User with login credentials.  😄

**Comment by dkniffin** [comment]
Yep, very true. I think I recently updated the seeds file and accidentally removed that part. We should create a user in `db/seeds/user.rb`
